src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -140,6 +140,24 @@ const std::vector<std::pair<ov::AnyMap, std::string>> autoExeDeviceConfigs = {
                                         ov::hint::performance_mode(ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT)}}, "CPU,GPU"),
             std::make_pair(ov::AnyMap{{ov::device::priorities(CommonTestUtils::DEVICE_GPU, CommonTestUtils::DEVICE_CPU),
                                         ov::hint::performance_mode(ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT)}}, "GPU,CPU"),
+            std::make_pair(ov::AnyMap{{ov::device::priorities(CommonTestUtils::DEVICE_GPU, CommonTestUtils::DEVICE_CPU),
+                                        ov::hint::performance_mode(ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT),
+                                        ov::hint::allow_auto_batching(true)}}, "GPU,CPU"),
+            #endif
+    };
+
+const std::vector<std::pair<ov::AnyMap, std::string>> multiExeDeviceConfigs = {
+            std::make_pair(ov::AnyMap{{ov::device::priorities("GPU.0")}}, "GPU.0"),
+            #ifdef ENABLE_INTEL_CPU
+            std::make_pair(ov::AnyMap{{ov::device::priorities(CommonTestUtils::DEVICE_GPU, CommonTestUtils::DEVICE_CPU)}}, "GPU,CPU"),
+            std::make_pair(ov::AnyMap{{ov::device::priorities(CommonTestUtils::DEVICE_CPU, CommonTestUtils::DEVICE_GPU)}}, "CPU,GPU"),
+            std::make_pair(ov::AnyMap{{ov::device::priorities(CommonTestUtils::DEVICE_CPU, CommonTestUtils::DEVICE_GPU),
+                                        ov::hint::performance_mode(ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT)}}, "CPU,GPU"),
+            std::make_pair(ov::AnyMap{{ov::device::priorities(CommonTestUtils::DEVICE_GPU, CommonTestUtils::DEVICE_CPU),
+                                        ov::hint::performance_mode(ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT)}}, "GPU,CPU"),
+            std::make_pair(ov::AnyMap{{ov::device::priorities(CommonTestUtils::DEVICE_GPU, CommonTestUtils::DEVICE_CPU),
+                                        ov::hint::performance_mode(ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT),
+                                        ov::hint::allow_auto_batching(true)}}, "GPU,CPU"),
```

Comment:
may be can you double confirm on multi GPU platform(e.g. GPU.0,GPU.1) with -d MULTI:GPU,CPU vs MULTI:GPU.0,CPU, in case we will meet unexpected failure if these kind of platform been introduced to CI?
I would expect:
MULTI:GPU,CPU -> GPU.0,GPU.1,CPU
AUTO:GPU,CPU with cumulative -> GPU.0,GPU.1,CPU
AUTO:GPU,CPU without cumulative -> first CPU then GPU.1 (assuming .1 is the dGPU)

Reply:
Okay